Dockerfile

```diff
@@ -21,53 +21,62 @@ RUN --mount=type=cache,target=/go/pkg/mod/ \
 # source code into the container.
 RUN --mount=type=cache,target=/go/pkg/mod/ \
     --mount=type=bind,target=. \
-<<EOF
+    <<EOF
     export VERSION_PKG_PATH="go.autokitteh.dev/autokitteh/internal/version"
     export TIMESTAMP="$(date -u "+%Y-%m-%dT%H:%MZ")"
-    export LDFLAGS="-X "${VERSION_PKG_PATH}.Version=$(cat .version || echo)" -X "${VERSION_PKG_PATH}.Time=${TIMESTAMP}" -X "${VERSION_PKG_PATH}.Commit=$(cat .commit || echo)""
+    export LDFLAGS="-s -w -X "${VERSION_PKG_PATH}.Version=$(cat .version || echo)" -X "${VERSION_PKG_PATH}.Time=${TIMESTAMP}" -X "${VERSION_PKG_PATH}.Commit=$(cat .commit || echo)""
     CGO_ENABLED=0 go build -o /bin/ak -ldflags="${LDFLAGS}" ./cmd/ak
 EOF
 
-################################################################################
-# Create a new stage for running the application that contains the minimal
-# runtime dependencies for the application.
+FROM python:3.12 AS python_deps
 
-FROM python:3.12-alpine3.20 AS final
+WORKDIR /app
 
-# Install any runtime dependencies 
-RUN --mount=type=cache,target=/var/cache/apk \
-    apk --update add \
-    ca-certificates \
-    tzdata \
-    && \
-    update-ca-certificates
+# RUN --mount=type=cache,target=/var/cache/apk \
+#     apt-get \
+#     g++ libstdc++ \
+#     ca-certificates \
+#     tzdata \
+#     && \
+#     update-ca-certificates
 
 COPY ./scripts/gcc-on-arm.sh /tmp
 RUN /tmp/gcc-on-arm.sh
 RUN rm /tmp/gcc-on-arm.sh
 
+
+RUN python3 -m venv env
```

Comment:
I'm not sure copying `venv` is the right thing, you can do it but it has risks. Why not skip this step and install on the final image? I don't think we have any dependencies that require compilation.

Reply:
it makes stuff goes much faster if you don't reinstall the packages on every go file change
and it reduces the final image size since you don't manage all the build tools required

Reply:
OK